tasks/main.yml

```diff
@@ -27,6 +27,11 @@
   when: elasticsearch_file_name|get_major_version < 5
   failed_when: "'ERROR' in plugin_response.stdout and 'already exists' not in plugin_response.stdout"
 
+- name: Install mapper-size plugin on ES5
+  shell: /usr/share/elasticsearch/bin/elasticsearch-plugin install mapper-size
```

Comment:
This will fail on second run. We need to make the action idempotent.

Also it'd be good to allow for it to not be installed, e.g. through a var that defaults to False.